guide/onboarding/protecting-a-wallet.md

```diff
@@ -0,0 +1,223 @@
+
+### What information is considered sensitive ?
+
+Sensitive information in wallet applications include the wallet balance, addresses, private keys and previous transactions information.
+- Wallet Balance - shows how much you own
+- Addresses - can be used to track your transaction history
+- Private keys - can be used to access and transfer your bitcoins
+
+It's more common for wallets to protect private keys, but not much is done for other sensitive information like the balance, addresses, and previous transactions. A few wallets like Bitcoin Core, Wasabi, Wallet of Satoshi, and others have made it work, though. Below is a pattern and considerations for hiding and revealing sensitive information.
+
+### Quickly hide from overview screen
+
+The hide icon/button, which is usually displayed within close reach of the balance itself, is used to quickly and easily hide and reveal wallet information by tapping or clicking on it.[^2]
+
+{% include picture.html
+   image = "/assets/images/guide/onboarding/protecting-a-wallet/hide-by-eye-icon.gif"
+   retina = "/assets/images/guide/onboarding/protecting-a-wallet/hide-by-eye-icon.gif"
+   mobile = "/assets/images/guide/onboarding/protecting-a-wallet/hide-by-eye-icon.gif"
+   mobileRetina = "/assets/images/guide/onboarding/protecting-a-wallet/hide-by-eye-icon.gif"
+   alt-text = "hide-by-eye icon"
+   width = 1600
+   height = 800
+%}
+
+#### Pros
+
+- It is quite convenient to switch between revealed and hidden state
+
+#### Cons
+
+- Easy for anyone else to reveal your information if they have access to your device.
+
+
+## Entering a PIN to reveal information
+
+A problem that seemed to arise throughout each implementation was that it was as easy to reverse the hidden state as it was to enable it. This may be for convenience sake, however, for protection against unauthorized access, perhaps you should only be able to unhide your information if a PIN or password has been entered. This could therefore reaffirm the identity of the wallet owner for extra security.
+
+{% include picture.html
+   image = "/assets/images/guide/onboarding/protecting-a-wallet/reveal-by-pin.gif"
+   retina = "/assets/images/guide/onboarding/protecting-a-wallet/reveal-by-pin.gif"
+   mobile = "/assets/images/guide/onboarding/protecting-a-wallet/reveal-by-pin.gif"
+   mobileRetina = "/assets/images/guide/onboarding/protecting-a-wallet/reveal-by-pin.gif"
+   alt-text = "reveal-by-PIN"
+   width = 1600
+   height = 800
+%}
+
+#### Pros
+
+- The risk of an unauthorized person revealing your information is minimal due to the PIN required
+
+#### Cons
+
+- It may not be convenient for the users to repeatedly put in their PIN when ever they want to reveal your information especially if they do so often.
```

Comment:
```suggestion
- It may not be convenient for users to repeatedly enter their PIN to reveal sensitive information, especially if they do this often.
```